External/Plugins/HaXeContext/Context.cs

```diff
@@ -1668,8 +1668,26 @@ public override bool BuildCMD(bool failSilently)
             RunCMD(command);
             return true;
         }
+
         #endregion
 
+        #region haxelib
+
+        internal void Install(List<string> libraries)
+        {
+            var haxePath = PathHelper.ResolvePath(GetCompilerPath());
+            if (!Directory.Exists(haxePath) && !File.Exists(haxePath))
+            {
+                ErrorManager.ShowInfo(TextHelper.GetString("Info.InvalidHaXePath"));
+                return;
+            }
```

Comment:
I would say that Haxelib detection should be extracted and centralized for this and `LookupLibrary`, also if done in a proper way, [Haxe Shim](https://github.com/lix-pm/haxeshim) or others could be supported as well.